tango_simlib/device_data_parser.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+###############################################################################
+# SKA South Africa (http://ska.ac.za/)                                        #
+# Author: cam@ska.ac.za                                                       #
+# Copyright @ 2018 SKA SA. All rights reserved.                               #
+#                                                                             #
+# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
+# WRITTEN PERMISSION OF SKA SA.                                               #
+###############################################################################
+"""This module performs the parsing of the TANGO device data json file ,
+containing the information needed to instantiate a useful device simulator.
+@author MeerKAT CAM team <cam@ska.ac.za>
+"""
+
+import logging
+import json
+
+from PyTango._PyTango import CmdArgType, AttrDataFormat
+
+MODULE_LOGGER = logging.getLogger(__name__)
+
+CMD_PROP_MAP = {
+    'name': 'name',
+    'in_type': 'dtype_in',
+    'in_type_desc': 'doc_in',
+    'out_type': 'dtype_out',
+    'out_type_desc': 'doc_out'
+}
+
+class DeviceDataParser(object):
```

Comment:
This is a very generic name - suggest we should refer to Fandango, since it is defining the file format.  Maybe `FandangoExportDeviceParser` or `FandangoExportParser`?
File name could be something like `fandango_json_parser.py`?